packages/opencensus-propagation-stackdriver/src/v1.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Does it make sense to update the copyright?

Reply:
I don't think so. This is derived from the source file in the Stackdriver trace agent.